lib/executable-config/lookup/obelisk-executable-config.cabal

```diff
@@ -8,6 +8,7 @@ library
   if os(ios)
     hs-source-dirs: src-ios
     build-depends: jsaddle-wkwebview
+    build-depends: directory
```

Comment:
Why does iOS need it but not Android?

Reply:
Android uses a special wrapper library to find resource files, but ios just treats them as files.